modules/simple_forms_api/app/models/simple_forms_api/vba_40_10007.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module SimpleFormsApi
+  class VBA4010007
+    include Virtus.model(nullify_blank: true)
+
+    attribute :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def metadata
+      {
+        'veteranFirstName' => @data.dig('name', 'first')
```

Comment:
Based on your JSON structure, I think this line would want to be:
```
'veteranFirstName' => @data.dig('application', 'applicant', 'name', 'first')
```
Or if it's supposed to be the `veteran` name as opposed to the `applicant` name you could do this:
```
'veteranFirstName' => @data.dig('application', 'veteran', 'service_name', 'first')
```

You'd then have to do that for all fields both on the metadata and on your `.json.erb` template.